analyser

```diff
@@ -193,21 +190,24 @@ def add(args):
         onvista_url = input.query_input("URL of onvista.de stock page",
                                         input.QueryType.URL)
 
-    qf_date = args.quarterly_figures_release_date
-    if not qf_date:
-        qf_date = input.query_input("Date of last quarterly figures release",
-                                    input.QueryType.DATE)
-
     s = Stock(sym)
-    s.quarterly_figure_dates.append(qf_date)
     s.onvista_fundamental_url = onvista_url
+    s.finanzen_net_url = args.finanzen_net_url
     s.update_stock_info()
     l = Levermann(s)
     l.evaluate()
     l.save()
     print(l)
 
 
+def oneshot(args):
+    s = Stock(isin=args.isin)
+    s.update_stock_info()
+    l = Levermann(s)
+    l.evaluate()
```

Comment:
We also need to call l.save() here.
The check if the file already exist and creating the data/ dir if it doesn't exist would also be good